.github/workflows/build.yml

```diff
@@ -25,8 +30,22 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_PULL_USERNAME }}
           password: ${{ secrets.DOCKER_PULL_PASSWORD }}
-      - name: Build image variant "${{ matrix.variant }}"
-        run: cd ${{ matrix.variant }} && docker buildx build ./ -t ${{ matrix.docker-tag }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          buildkitd-flags: --debug
+
+      - name: Build and push image for "${{ matrix.variant }}"
+        uses: docker/build-push-action@v3
+        with:
+          context: ${{ matrix.variant }}
+          platforms: "linux/arm/v6,linux/arm/v7,linux/s390x,linux/ppc64le,linux/386,linux/amd64,linux/arm64/v8"
+          push: true 
+          tags: ${{ matrix.docker-tag }}
+          # does not work linux/arm/v5 AND linux/mips64le - composer does not support  mips64le or armv5 although docker buildx does support them
```

Comment:
```suggestion
          # does not work linux/arm/v5 AND linux/mips64le - composer does not support  mips64le or armv5 nor does the php image support them on the alpine variant
```

So the two are supported only on PHP non Alpine images, see: https://hub.docker.com/_/php/tags?page=1&name=8.0-fpm
See: https://hub.docker.com/_/php/tags?page=1&name=8.0-fpm-alpine

Reply:
For reference:

PHP:
![image](https://user-images.githubusercontent.com/7784660/212985828-2f28ae8e-1e2a-4411-b3be-6bb9de81ae7b.png)
![image](https://user-images.githubusercontent.com/7784660/212985854-536ed956-2b5b-4708-baae-68d90a457179.png)

Composer:
![image](https://user-images.githubusercontent.com/7784660/212986013-9cf07e1b-39ec-4c09-94e9-431a82ce8549.png)